docker-compose.yaml

```
@@ -103,6 +103,49 @@ services:
       retries: 5
     volumes:
       - agent-ssh-dir:/home/jenkins/.ssh:ro  # Mounts the agent-ssh-dir volume to the /home/jenkins/.ssh path inside the container as read-only
+
+
+  multi_jenkins_controller:
+    image: ${DOCKERHUB_USERNAME}/jenkinsci-tutorials:multi_controller_end-to-end-multibranch
```

```suggestion
    image: ${DOCKERHUB_USERNAME}/jenkinsci-tutorials:multi_controller_end-to-end-multibranch_
```

I think, currently have _ in the end, right?